lib/jsrt/JsrtSourceHolder.cpp

```diff
@@ -49,7 +51,6 @@ namespace Js
             *utf8Script = HeapNewArray(utf8char_t, cbUtf8Buffer);
         }
 
-        Assert(length < MAXLONG);
```

Comment:
From the documentation perspective, it is very hard to understand the limit there. Combination of  `assert` and `if`statements are already broken on 32/64. 

Current `if` statement doesn't work on 32 while `Assert` doesn't work on 64.